src/metabase/task/sync_cache_analyze_classify_databases.clj

```diff
@@ -0,0 +1,101 @@
+(ns metabase.task.sync-cache-analyze-classify-databases
```

Comment:
This should have a simple/shorter name. 
"Analyze Databases" or "Collect Metadata" perhaps.